src/main/webapp/app/entities/course/course.route.ts

```diff
@@ -50,6 +51,18 @@ export const courseRoute: Routes = [
         },
         canActivate: [UserRouteAccessService]
     },
+    {
+        path: 'course/:courseId',
+        component: CourseExercisesOverviewComponent,
+        resolve: {
+            course: CourseResolve
+        },
+        data: {
+            authorities: ['ROLE_INSTRUCTOR', 'ROLE_ADMIN'],
```

Comment:
Is this the route to see the new exercise overview? If yes, tutors should also see it